pkgs/top-level/perl-packages.nix

```diff
@@ -5439,6 +5456,19 @@ let
     };
   };
 
+  DataStag = buildPerlPackage {
+    pname = "Data-Stag";
+    version = "0.14";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/C/CM/CMUNGALL/Data-Stag-0.14.tar.gz";
+      sha256 = "4ab122508d2fb86d171a15f4006e5cf896d5facfa65219c0b243a89906258e59";
+    };
+    propagatedBuildInputs = [ IOString ];
+    meta = {
+      description = "Structured Tags";
+    };
```

Comment:
Missing a `license` here. Likely:
```      
license = with lib.licenses; [ artistic1 gpl1Plus ];
```

Since its POD says "This module is free software. You may distribute this module under the same terms as perl itself"